pkg/cloudsources/paladin/client.go

```diff
@@ -208,23 +209,36 @@ func NewClient(cfg *storage.CloudSource, options ...opts.ClientOpts) *paladinCli
 	retryClient.RetryWaitMin = 10 * time.Second
 
 	return &paladinClient{
-		httpClient:    retryClient.StandardClient(),
-		endpoint:      urlfmt.FormatURL(cfg.GetPaladinCloud().GetEndpoint(), urlfmt.HTTPS, urlfmt.NoTrailingSlash),
-		cloudSourceID: cfg.GetId(),
+		httpClient:      retryClient.StandardClient(),
+		endpoint:        urlfmt.FormatURL(cfg.GetPaladinCloud().GetEndpoint(), urlfmt.HTTPS, urlfmt.NoTrailingSlash),
+		cloudSourceID:   cfg.GetId(),
+		cloudSourceName: cfg.GetName(),
 	}
 }
 
 func (c *paladinClient) Ping(ctx context.Context) error {
 	// At the current time, no better API is known besides the asset API to confirm correct authN/Z setup and
 	// connectivity. In case we find a better API, we may switch to that.
-	_, err := c.getAssets(ctx)
-	return err
+	if _, err := c.getAssets(ctx); err != nil {
+		log.Errorw("Getting assets",
+			logging.Err(err),
+			logging.ErrCode(codes.PaladinCloudGeneric),
+			logging.CloudSourceName(c.cloudSourceName),
+		)
+		return err
+	}
+	return nil
 }
 
 // GetDiscoveredClusters returns the discovered clusters from the Paladin Cloud API.
 func (c *paladinClient) GetDiscoveredClusters(ctx context.Context) ([]*discoveredclusters.DiscoveredCluster, error) {
 	response, err := c.getAssets(ctx)
 	if err != nil {
+		log.Errorw("Getting assets",
```

Comment:
We should consider using the same messages in wrapped errors that we output to console and administrative errors in the UI.

In this example, we should use `"retrieving data from paladin cloud"` in both places or the same wording.

The same comment can be applied to:
```
			log.Errorw("Getting cluster subscription",
				logging.Err(err),
				logging.ErrCode(codes.OCMCloudGeneric),
				logging.CloudSourceName(c.cloudSourceName),
			)
			return nil, pkgErrors.Wrap(err, "retrieving cluster subscriptions")
```
in `pkg/cloudsources/ocm/client.go`